src/styles/block/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+  }
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when selectors are comma-separated, each selector should be on a new line for better readability.